src/Tools/dotnet-monitor/CollectionRules/Options/Actions/LoadProfilerOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Diagnostics.Monitoring.WebApi.Models;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Actions
+{
+    /// <summary>
+    /// Options for the <see cref="CollectionRules.Actions.LoadProfilerAction"/> action.
+    /// </summary>
+    [DebuggerDisplay("LoadProfiler")]
+    internal sealed partial class LoadProfilerOptions
```

Comment:
Please add additional tests that verify the validation of the properties. See https://github.com/dotnet/dotnet-monitor/blob/main/src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/CollectionRuleOptionsTests.cs for examples of verifying the minimally required properties that need to be set to pass validation (*_MinimumOptions), round tripping testing (*_RoundTrip), and full validation failure validation (*_PropertyValidation).